keyboards/preonic/keymaps/boy314/config.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CONFIG_USER_H
```

Comment:
If you could replace the first two lines with just `#pragma once`, and remove the endif at the end?